c-deps/libroach/row_counter.cc

```diff
@@ -91,6 +91,7 @@ bool RowCounter::Count(const rocksdb::Slice& key, cockroach::roachpb::BulkOpSumm
   if (decoded_key.data() == prev_key.data) {
     return true;
   }
+  free(prev_key.data);
```

Comment:
I’m pretty sure prev_key should be a std::string. Look at the comparison just above. That isn’t doing what the author thought it was doing.

Reply:
Yeah... You're right... I'm totally unfamiliar with this code base.
Anyways, Fixed the comparison plus added row counter destructor to avoid leaking the last "prev_key" memory